src/test/java/redis/clients/jedis/tests/JedisTest.java

```diff
@@ -188,4 +188,56 @@ public void testBinaryBitfield() {
     }
   }
 
+  @Test
```

Comment:
We're placing command tests to other places. I guess you referred above tests but above tests (BitField and BinaryBitField) are also wrong, and so it should be moved, too.